code/datums/ammo/ammo.dm

```diff
@@ -173,6 +173,8 @@
 		living_mob.apply_stamina_damage(fired_projectile.ammo.damage, fired_projectile.def_zone, ARMOR_BULLET)
 
 /datum/ammo/proc/slowdown(mob/living/living_mob, obj/projectile/fired_projectile)
+	if(living_mob.mob_size >= MOB_SIZE_BIG)
```

Comment:
some t2s are `MOB_SIZE_BIG`
consider instead type checking to ensure its a xeno and then check `tier`.

Reply:
Sure but slugs for example don't affect carriers, hivelords etc.